### DISCUSSION TITLE
Sonoff TH01 - "Configure failed....(Route error)"

### DISCUSSION THREAD

**Discussion by dwagix** [Koenkk/zigbee2mqtt#14849] [Devices]
Hi All,

I have four TH01 Sonoff Temperature & Humidity sensors which were all working fine for months...when suddenly one stopped responding.  I attempted to remove/re-add, and am met with:

Error 2022-11-06 20:42:22 Configure failed: Error: Bind 0x00124b00251fae21/1 msTemperatureMeasurement from '0x84b4dbfffefcdf23/1' failed (Route error)

The interview completes ("Successfully interviewed / device has successfully been paired") and it joins to z2m but doesn't expose anything except the LQI (temp/humidity are N/A)

I'm running a Sonoff Dongle-E with the latest-dev branch - which I updated to current tonight (5e2df18) - but no luck.

I've reset, restarted, pulled the battery - several times over the last week...always with the same results

Open to any suggestions - thanks in advance!

**Comment by og-gh** [comment]
I have the same problem with one of my 8 sensors.
I'm running 1.28.2.
It might be a regression of [#8971].

**Reply by og-gh** [reply]
I followed these steps and it worked for me:

1. Force remove device.
2. Pair the device as per usual process. It should pair successfully, but device configuration is not completed (e.g. question mark where Power property should be in device list, "Exposes" shows N/A).
3. NOW remove battery, wait for 5-10 seconds
4. Re-insert battery. The device will rejoin the network
5. Zigbee2Mqtt should finalise the configuration process successfully,
6. "Exposes" should now show correct temp/hum/batt values

**Comment by cipnt** [comment]
I had the exact problem with one of my TH01 sensors. 
Initially I thought it had kicked the bucked; I had it for a few years so it wouldn't have been a shock.
But I tried pairing it near the coordinator and that fixed it, it now works as before even after moving it back to its original location. 
This was strange since most advice is to pair zigbee devices in their approximate final location so that they discover their neighbours and establish a route to the coordinator.
Hope this helps someone...
